include/openssl/txt_db.h

```diff
@@ -27,7 +27,6 @@
 # define DB_ERROR_INDEX_OUT_OF_RANGE     3
 # define DB_ERROR_NO_INDEX               4
 # define DB_ERROR_INSERT_INDEX_CLASH     5
-# define DB_ERROR_WRONG_NUM_FIELDS       6
```

Comment:
This is an API break and so it would not be acceptable earlier than in 4.0.

Why do you remove this particular error?

Reply:
Ah, it isn't really returned anywhere. But then OK, just keep the define here and perhaps add a comment, that it is never really returned.

Reply:
I'll fix. Thanks!